documentation/synthetic_tokens/tutorials/liquidation.md

```diff
@@ -0,0 +1,423 @@
+
+# Start the truffle console
+truffle console --network kovan_mnemonic
+
+# Print the address of your newly created account
+accounts[0]
+0x45Bc98b00adB0dFe16c85c391B1854B706b7d612
+```
+
+You can now fund this wallet with the associated currency for the type of bot you want to run.
+To learn more about creating synthetic tokens to back fund your liquidator bot see [this](./using_the_uma_sponsor_cli_tool.md) tutorial.
+
+### Creating a price feed API key
+
+All bots require a price feed to inform their liquidation decisions.
+The easiest price feed to intergrade into is [CryptoWatch](https://cryptowat.ch/). To create an API Key do the following:
+
+1. Create an account [here](https://cryptowat.ch/account/create).
+2. Generate an API key [here](Generate an API key).
+
+Keep this key handy. You'll need it when configuring the bots.
+
+## Running the liquidator and disputer bots locally
+
+**a) Clone repo & install dependencies**
+
+Run the following from the root directory to install dependencies and compile the contracts:
+
+```bash
+# Clone the repo and navigate into the protocol directory
+git clone https://github.com/UMAprotocol/protocol.git
+cd ./protocol
+
+# Install dependencies & compile the contracts
+npm install
+truffle compile
+```
+
+**b) Configuring environment**
+
+The bots can be easily run directly from your local development environment. To start a bot the first step is to configure the bot's settings.
+Liquidation bots require 4 main configurations settings which are configured using environment variables.
+To set this up create a `.env` file in the `/core` directory of the repo it:
+
+```bash
+POLLING_DELAY=30000
+EMP_ADDRESS=0xDe15ae6E8CAA2fDa906b1621cF0F7296Aa79d9f1
+MNEMONIC=sail chuckle school attitude symptom tenant fragile patch ring immense
+PRICE_FEED_CONFIG={"type":"medianizer","apiKey":"YOUR_API_KEY","pair":"ethbtc","lookback":7200,"minTimeBetweenUpdates":60,"medianizedFeeds":[{"type":"cryptowatch","exchange":"coinbase-pro"},{"type":"cryptowatch","exchange":"binance"},{"type":"cryptowatch","exchange":"bitstamp"}]}
+```
+
+The parameters above, as well as other optional parameters are explained in the appendix of this tutorial. **Be sure to add in your mnemonic and your crypto watch API key.** The parameter in the example above conform to [UMIP-2](<[../..](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-2.md#implementation)>)'s specification.
+
+**c) Starting the bots**
+
+Now that your env is set up you can run the bot. Run the following command from the `core` directory to start the bots on Kovan:
+
+```bash
+truffle exec ../liquidator/index.js --network kovan_mnemonic
+```
+
+This will start the liquidator bot process using the network `kovan` and the wallet `mnemonic`. If you have any issues executing the `truffle` command you can try running `npx truffle <command>` or `$(npm bin)/truffle <command>`. You should see the following output:
```

Comment:
nit: if we want to default to not using `$(npm bin)`, we should put this caveat way above in the tutorial IMO.

Reply:
bump

Reply:
Cool. I moved this up into a new pre-req section where I discuss installing the dependencies.